pull_request_template.md

```diff
@@ -0,0 +1,32 @@
+# PULL REQUEST
+
+<!--
+Please read and fill out this form before submitting your PR.
+
+Please make sure you have reviewed our contributors guide before submitting your
+first PR.
+-->
+
+## Overview
+
+<!-- 
+Please provide an explanation of the PR, including the apprioprate context,
+background, goal, and rationale. If there is an issue with this information,
+please provide a tl;dr and link the issue. 
+-->
+
+## Checklist
+
+<!-- 
+Please complete the checklist to ensure that the PR is ready to be reviewed.
+
+IMPORTANT:
+PRs should be left in Draft until the below checklist is completed.
+-->
+
+- [ ] New and updated code has appropriate documentation
+- [ ] New and updated code has new and/or updated testing
+- [ ] PR is linked to appropriate project
```

Comment:
I don't think we had this discussion before. Ideally, all PRs should be linked to Issues (because work is planned ahead, etc). To avoid total spam of boards, my recommendation is to assign Issues to project/sprint/milestone/etc, but keep PRs just linked to corresponding issue. This can be easily displayed on project boards and data is not duplicated.

```suggestion
- [ ] PR is linked to corresponding issue(s)
```

Reply:
I'll just remove this then